ServiceTier/Services/SignUpService.cs

```diff
@@ -36,7 +36,10 @@ public User GetUserById(int id)
 
         private bool UserExists(UserDTO model)
         {
-            var userExist = unitOfWork.UserManager.GetAll().Any(u => u.Email == model.Email);
+            bool userExist = false;
+            var userToFind = unitOfWork.UserManager.FindByEmailAsync(model.Email).Result;
+            if (userToFind != null)
+                userExist = true;
 
             return userExist;
```

Comment:
return userToFind != null;
or return unitOfWork.UserManager.FindByEmailAsync(model.Email).Result != null

Reply:
Changed to userToFind != null;